sentry-core/src/main/java/io/sentry/core/Session.java

```diff
@@ -0,0 +1,99 @@
+package io.sentry.core;
+
+import java.util.Date;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public final class Session {
+
+  public enum State {
+    // TODO: What about 'Started' 'Ended'? Ok is not very clear
+    Ok,
+    Exited,
+    Crashed
```

Comment:
let's make all uppercase, so its easier to cast json<-> enum, otherwise we would need to `capitalize`, we also keep the same pattern as `SentryLevel`